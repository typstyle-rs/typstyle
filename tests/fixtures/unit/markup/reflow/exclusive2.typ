/// typstyle: wrap_text

To this neural network, we also add another convex term:
$
  [frac(abs(x_i)^k x_i, ell)]_i
$ <eqn:odd_power_term>
