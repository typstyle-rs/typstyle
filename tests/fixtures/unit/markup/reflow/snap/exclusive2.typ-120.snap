---
source: tests/src/unit.rs
input_file: tests/fixtures/unit/markup/reflow/exclusive2.typ
---
/// typstyle: wrap_text

To this neural network, we also add another convex term:
$
  [frac(abs(x_i)^k x_i, ell)]_i
$ <eqn:odd_power_term>

Another equation$ "no space around!" $aha! Another equation$ "labeled" $<eqn:tricky>!
